sentry/content.md

```diff
@@ -8,19 +8,41 @@ Sentry is a realtime event logging and aggregation platform. It specializes in m
 
 # How to use this image
 
-## start a sentry instance
+## how to setup a full sentry instance
 
-### PostgreSQL database (as recommended by upstream)
+1.	start a redis container
 
-	docker run --name some-sentry --link some-postgres:postgres -d sentry
+		docker run -d --name some-redis redis
 
-### MySQL database
+2.	start a MySQL or PostgreSQL(recommended by upstream) database container (later steps assume PostgreSQL, replace the `--link` entries appropriately for MySQL)
 
-	docker run --name some-sentry --link some-mysql:mysql -d sentry
+		# postgres:
+		docker run -d -e POSTGRES_PASSWORD=secret -e POSTGRES_USER=sentry --name some-postgres postgres
+		# or mysql:
+		docker run -d -e MYSQL_ROOT_PASSWORD=secret -e MYSQL_DATABASE=sentry --name some-mysql mysql
 
-### Redis buffering (recommended by upstream for any real workloads)
+3.	now start up sentry server
 
-To enable Update Buffers using Redis, just add `--link some-redis:redis` to the `docker run` arguments of your service.
+		docker run -d --name some-sentry --link some-redis:redis --link some-postgres:postgres sentry
+		# or
+		docker run -d --name some-sentry --link some-redis:redis --link some-mysql:mysql sentry
```

Comment:
You already told me these steps would assume `postgres`, so I'd say let's axe this second option for simplicity.